src/main/java/org/spongepowered/api/statistic/achievement/Achievement.java

```diff
@@ -24,34 +24,19 @@
  */
 package org.spongepowered.api.statistic.achievement;
 
-import org.spongepowered.api.CatalogType;
-import org.spongepowered.api.Sponge;
+import org.spongepowered.api.item.inventory.ItemStack;
 import org.spongepowered.api.statistic.Statistic;
-import org.spongepowered.api.text.translation.Translatable;
 import org.spongepowered.api.text.translation.Translation;
-import org.spongepowered.api.util.ResettableBuilder;
 import org.spongepowered.api.util.annotation.CatalogedBy;
 
 import java.util.Collection;
 import java.util.Optional;
 
-import javax.annotation.Nullable;
-
 /**
  * Represents an in-game achievement which may be earned by or given to players.
  */
 @CatalogedBy(Achievements.class)
-public interface Achievement extends CatalogType, Translatable {
-
-    /**
-     * Creates a new {@link Builder} to build a {@link Achievement}.
-     *
-     * @return The new builder
-     */
-    static Builder builder() {
-        return Sponge.getRegistry().createBuilder(Builder.class);
-    }
-
+public interface Achievement extends Statistic {
 
```

Comment:
Essentially yes. However, I did implement it in such a way that they are always separated by the API. `StatData` will not contain `Achievements` and `AchievementData` will not contain `Statistics`. I think this is fine since there is no actual value in a `Statistic` and all the other implemented methods still make sense for an `Achievement`.